pages/integrations/sso/okta.md.erb

```diff
@@ -24,12 +24,32 @@ In your Buildkite Organization Settings' Single Sign On menu item, choose the Ok
 
 On the following screen in the Metadata URL field, enter the 'Identity Provider Metadata' link that you copied in the previous step.
 
-<%= image "okta-settings.png", width: 1848/2, height: 732/2, alt: "Screenshot of the Buildkite Okta Settings Page" %>
+<%= image "okta-settings.png", width: 1742/2, height: 606/2, alt: "Screenshot of the Buildkite Okta Settings Page" %>
 
 <div class="Docs__note">
   <p>You can also set up SSO providers manually with GraphQL. See the <a href="/docs/integrations/sso/sso-setup-with-graphql">SSO Setup with GraphQL Guide</a> for detailed instructions and code samples.</p>
 </div>
 
+### SCIM Deprovisioning
+
+Using the provisioning settings in Okta you can enable automatic deprovisioning of users in your Buildkite organization.
```

Comment:
Yeah! I'm going to leave off 'add' because we're not doing that with the new SCIM bit, it's just the deprovisioning part we're adding. Add users will still happen with the OG SAML. I'll swap it to your words for the rest 🙌🏻